src/CourseAuthoringPage.jsx

```diff
@@ -19,17 +23,28 @@ export default function CourseAuthoringPage({ courseId, children }) {
   const courseNumber = courseDetail ? courseDetail.number : null;
   const courseOrg = courseDetail ? courseDetail.org : null;
   const courseTitle = courseDetail ? courseDetail.name : courseId;
+  const courseAppsApiStatus = useSelector(getCourseAppsApiStatus);
+  const inProgress = useSelector(getLoadingStatus) === RequestStatus.IN_PROGRESS;
+  if (courseAppsApiStatus.status === RequestStatus.DENIED) {
+    return (
+      <PermissionDeniedAlert />
+    );
+  }
+
+  const AppHeader = () => (
+    <Header
+      courseNumber={courseNumber}
+      courseOrg={courseOrg}
+      courseTitle={courseTitle}
+      courseId={courseId}
+    />
+  );
 
   return (
     <>
-      <Header
-        courseNumber={courseNumber}
-        courseOrg={courseOrg}
-        courseTitle={courseTitle}
-        courseId={courseId}
-      />
+      {inProgress ? <Loading /> : AppHeader()}
       {children}
-      <Footer />
+      {!inProgress ?? <Footer />}
```

Comment:
https://github.com/edx/frontend-app-course-authoring/pull/153#discussion_r665691083
what I meant was 
{!inProgress & <Footer />} I am not sure if we have `??` this operator